spec/factories/measurements.rb

```diff
@@ -1,9 +1,9 @@
 FactoryBot.define do
   factory :measurement do
-    name { "MyString" }
-    value_type { "MyString" }
     value { "" }
+    measurement_type
+    association :subject, factory: :tank
```

Comment:
I think `:animal` will likely end up being the default we want here, and we will likely add traits for the rest of the possibilities in the polymorphic association.